include/click/dpdkdevice.hh

```diff
@@ -76,20 +87,51 @@ private:
         unsigned n_tx_descs;
     };
 
+    DevInfo info;
```

Comment:
All access inside class functions would change with this change. So it would ruin my changeset^^